internal/heartbeat/heartbeat.go

```diff
@@ -5,14 +5,13 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"os"
 	"sync"
 	"sync/atomic"
 	"time"
 
-	"git.sr.ht/~spc/go-log"
 	"github.com/go-openapi/strfmt"
 	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
```

Comment:
Question: do we need to set the formatter once again?

Reply:
No, no need to do that at all.